public/index.html

```diff
@@ -0,0 +1,28 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List In JavaScript</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css" />
+    <link rel="stylesheet" href="../src/style.css">
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="new-todo">
+          <form id="todoform">
+            <input type="text" name="newtodo" id="newtodo" placeholder="I will do ..." />
+          </form>
```

Comment:
- After cloning your repo and running the server, this form is the only thing I can see on the browser. Can you kindly fix this? Or better still, can you ensure to update your Readme with steps on how to install and run your project locally. If you like, I think you can also consider making a loom video showing how your project functions as well.

This is what i see when i run the server 👇🏽 

![todobd](https://user-images.githubusercontent.com/99246359/206857894-e92244b5-4071-4c95-a9cd-d4e21ebae796.PNG)

- Also, kindly ensure that all todos added are being saved in the Local storage. 🌸